packages/node/src/integrations/hapi.ts

```diff
@@ -0,0 +1,163 @@
+import type { Boom } from '@hapi/boom';
```

Comment:
We should not/cannot use types like these, as then we need to add hapi as a dependency 😬 let's just inline the types we need in here!

Reply:
Done 👍  [21beab3](https://github.com/getsentry/sentry-javascript/pull/9539/commits/21beab31e7b8cb3eeff2a83bc38a41b4cc38ca55)